src/app/shared/header/header.component.scss

```diff
@@ -0,0 +1,18 @@
+ul li {
```

Comment:
try to have this styling in the global incase you want to implement to all the styling usually we have this in the global stylesheet